DataConnectors/OnapsisOP.json

```diff
@@ -0,0 +1,240 @@
+{
+
+    "id": "OnapsisOP",
+
+    "title": "Onapsis OP",
```

Comment:
Can you give full form in the title (OP)? Is the appliance name OSP or OP?

Reply:
Currently in the logs, the appliance is defined as OSP but our appliance is called OP (Onapsis Platform). Should I change it to "Onapsis OSP' or "Onapsis Platform"?

Reply:
- Will fix the usage instructions in the parser, thanks for pointing that out
- The incident_lookup function is used at line 322 in OnapsisAlarmsOverview.json

Reply:
Currently in the logs, the appliance is defined as OSP but our appliance is called OP (Onapsis Platform). Should I change it to "Onapsis OSP' or "Onapsis Platform"?

Reply:
Onapsis Platform will be meaningful without abbreviations.

Reply:
Just committed the changes, let me know what you think

Reply:
Could you please provide the usage of "incident_lookup"- Its not used in workbook as mentioned in the Kusto function definition. Where is this function used?

Reply:
If you take a look at line 322 of the workbook file (OnapsisAlarmsOverview.json) - incident_lookup is used in a query - it's goal being to provide context into what the incident means.